### PR TITLE
Added translation into Russian

### PR DESCRIPTION
**Pull request by VinAdmin**
Please approve

### CODE REVIEW

File: locale/ropes.ru.tr [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+rope boxes can hold @1m of rope.=Ящики для веревок вмещают до 1 м веревки.
+rope boxes can hold rope lengths from @1m to @2m.=Ящики для веревок могут вмещать веревки длиной от 1 до 2 метров.
```
Comment (SmallJoker) [inline locale/ropes.ru.tr]: Incorrect `@n` notation in the translated text.

**Comment by SmallJoker** [comment]
> #WARNING: AUTOTRANSLATED BY GOOGLE TRANSLATE

Did you need assistance to understand the English sentence, or did you mostly copy the Russian translation from Google into this file?

---

Native speakers are welcome to have a quick look at the translation before merge.

File: locale/ropes.ru.tr (hunk 1 above)
Comment (VinAdmin) [reply locale/ropes.ru.tr]: fixed it [9304d14](https://github.com/minetest-mods/ropes/pull/31/commits/9304d149e54fd50ba49acd5247c7196e81afde6c)

**Comment by VinAdmin** [comment]
> > #WARNING: AUTOTRANSLATED BY GOOGLE TRANSLATE
> 
> Did you need assistance to understand the English sentence, or did you mostly copy the Russian translation from Google into this file?
> 
> Native speakers are welcome to have a quick look at the translation before merge.

Yes, to understand and adjust the meaning.

**Pull request merged**